codevalidator-api/src/main/java/org/sitenv/vocabularies/engine/ValidationEngine.java

```diff
@@ -807,10 +681,6 @@ public void run() {
 			{
 				logger.error("Failed to load configured vocabulary directory.", e);
 			}
-			
-			// TODO: Perform Validation/Verification, if needed
-			Runtime.getRuntime().gc();
```

Comment:
- Manual GCs don't help for work like this (CPU-bound + IO-bound work).
- The new default GC (G1) is already quite good at automatically performing quick cycles without blocking.